packages/docs/components/CodePreview/CodePreview.tsx

```diff
@@ -31,11 +32,13 @@ function getInitialCode(children: React.ReactNode, language: Language): string {
     return children;
   }
 
-  const code = transform(children, {
+  const transformResult: BabelFileResult = transform(children, {
```

Comment:
We should be able to get rid of this.
```suggestion
  const transformResult = transform(children, {
```